x-pack/plugins/security_solution/public/common/components/sourcerer/readme.md

```diff
@@ -0,0 +1,53 @@
+# Sourcerer Component
+
+
+### Adding sourcerer to a new page
+- In order for the sourcerer to show up on a page, it needs to be added to the array `sourcererPaths` in `containers/sourcerer/index.tsx`
+- The scope of a sourcerer component will be default unless the path is added to the `detectionsPaths` array, in which case the scope can be detections in `containers/sourcerer/index.tsx`
+
+## Default Sourcerer
+![](../../images/default.png)
+- The data view is shown in the dropdown. All index aliases that have existing indices are selected by default. Index aliases that do NOT have existing indices are shown in the dropdown as deactivated
+
+## Detections Sourcerer
+![](../../images/alerts.png)
+- The detections sourcerer is locked to the signals index, and has an "Alerts" flag
```

Comment:
On the alerts page, do we still load all the data view index fields or just the selected index (the alerts index in this case)?

Reply:
yes, all of the index fields are loaded by the data view id, not according to the index pattern